python/lsst/pipe/tasks/quickFrameMeasurement.py

```diff
@@ -517,4 +559,13 @@ def _run(self, exp, doDisplay=False):
         result.brightestObjApFlux70 = brightestObjApFlux70
         result.brightestObjApFlux25 = brightestObjApFlux25
         result.medianXxYy = medianXxYy
+
+        result.brightestObjCentroidCofM = None
+        try:
+            boxSize = donutDiameter * 1.3  # allow some slack, as cutting off side of donut is very bad
```

Comment:
If missing the side is bad, should you check for that, or will you just get an exception and return None, indicating that there was a problem?  I'm also not clear what would cause this block to raise.  _getCenterOfMass doesn't directly raise, and I don't see why anything else would raise either.  Am I just missing something obvious>

Reply:
It's not that bad, it just means the result won't be very good, and checking for it is definitely not easy (all of which we discussed OOB, but just noting here too). Also, downstream code with see that, and it's not a huge problem anyway (plus that code is responsible for passing the boxsize anyway).

You're right about the `try`, and I've removed it completely now (having run some tests to make sure I'm definitely OK if the boxsize goes off the edge of the chip).